web/src/app/rotations/RotationAddUserDialog.js

```diff
@@ -12,66 +11,54 @@ const mutation = gql`
   }
 `
 
-export default class RotationAddUserDialog extends React.Component {
-  static propTypes = {
-    rotationID: p.string.isRequired,
-    userIDs: p.array.isRequired,
-    onClose: p.func.isRequired,
+const RotationAddUserDialog = (props) => {
+  const { rotationID, onClose } = props
+  const [value, setValue] = useState(null)
+  const defaultValue = {
+    users: [],
   }
+  // append to users array from selected users
+  const users = []
+  const userIDs = (value && value.users) || defaultValue.users
+  props.userIDs.forEach((u) => users.push(u))
+  userIDs.forEach((u) => users.push(u))
 
-  state = {
-    value: null,
-    errors: [],
-  }
-
-  render() {
-    const defaultValue = {
-      users: [],
-    }
-
-    return (
-      <Mutation mutation={mutation}>
-        {(commit, status) => this.renderDialog(defaultValue, commit, status)}
-      </Mutation>
-    )
-  }
+  const [updateRotationMutation, updateRotationMutationStatus] = useMutation(
```

Comment:
```suggestion
  const [updateRotationMutation, { loadng, error, data }] = useMutation(
```

We can inline loading, error, and data here so we don't have to call a long var name later on